components/discord_bot/utils.mjs

```diff
@@ -118,4 +118,13 @@ export default {
       ? undefined
       : value;
   },
+  parseObject(obj) {
+    let parsed;
+    try {
+      parsed = JSON.parse(obj);
+    } catch (e) {
+      parsed = obj;
+    }
+    return parsed;
+  },
```

Comment:
Move this file to `common/utils.mjs`

Reply:
@lcaresia this may result to refactor all other actions and sources. How about letting it stay where it is? What do you think?

Reply:
Yes, because when a common file is not in folder `common` or start with word `common`, will result in a error on deploy to production.

I thing the best way is move this file to `common` folder, refactor the other actions/sources and upgrade they versions.